### ISSUE TITLE
[BUG] add_column reports error

### ISSUE THREAD

**Issue by wuyueandrew** [alibaba/GraphScope#1728]
**Describe the bug**
Demo k_core reports error

**To Reproduce**
code:
`
g = load_ogbn_mag(sess, "/testingdata/ogbn_mag_small/")
ret1 = graphscope.k_core(g, k=5)
g = g.add_column(ret1, {"kcore": "r"})

`
error_log:
2022-06-16 17:04:21,490 [ERROR][rpc:187]: Runstep failed with code: ANALYTICAL_ENGINE_INTERNAL_ERROR, message: Error occurred during preprocessing, The traceback is: Traceback (most recent call last):
  File "/home/graphscope/.local/lib/python3.8/site-packages/gscoordinator/coordinator.py", line 426, in run_on_analytical_engine
    for response in responses:
  File "/home/graphscope/.local/lib/python3.8/site-packages/grpc/_channel.py", line 426, in __next__
    return self._next()
  File "/home/graphscope/.local/lib/python3.8/site-packages/grpc/_channel.py", line 826, in _next
    raise self
grpc._channel._MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.INTERNAL
	details = "/work/analytical_engine/core/server/dispatcher.cc:99: operator() -> Unmatched std::exception detected: [json.exception.type_error.305] cannot use operator[] with a string argument with number
void boost::throw_exception<boost::property_tree::json_parser::json_parser_error>(boost::property_tree::json_parser::json_parser_error const&, boost::source_location const&) + 0x9D6
gs::Dispatcher::publisherLoop() + 0x1C7
execute_native_thread_routine + 0x10
start_thread + 0xD9
clone + 0x43
**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - GraphScope version: [0.14.0]
 - OS: [Linux]
 - Kubernetes Version [1.19]

**Additional context**

**Comment by sighingnow** [comment]
The error message here is quite hard to parse for humans and needs to be fixed.

**Comment by wuyueandrew** [comment]
> The error message here is quite hard to parse for humans and needs to be fixed.

maybe add the reason to faq is a simple way
